tests/test_ddebug_canc/test.log

```diff
@@ -0,0 +1,12 @@
+make[4] : on entre dans le répertoire « /home/alan/Bureau/Lib/verificarlo/tests/test_ddebug_canc »
```

Comment:
`tests/test_ddebug_canc/test.log` should not be committed since it is generated when the tests are run.